osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -665,4 +492,24 @@ public enum ScoringMode
         [LocalisableDescription(typeof(GameplaySettingsStrings), nameof(GameplaySettingsStrings.ClassicScoreDisplay))]
         Classic
     }
+
+    [Serializable]
+    [MessagePackObject]
+    public class ScoreProcessorStatistics
+    {
+        [Key(0)]
+        public double MaximumBaseScore { get; set; }
+
+        [Key(1)]
+        public double BaseScore { get; set; }
+
+        [Key(2)]
+        public int CountAccuracyJudgements { get; set; }
```

Comment:
This kind of variable needs xmldoc. Also maybe needs to be renamed, once I understand what it's meant to be (I think `AccuracyJudgementCount` might be more correct?)

This change removed xmldoc on other properties in this class, see above in the diff.